code/AssetLib/glTF/glTFImporter.cpp

```diff
@@ -80,7 +80,7 @@ static const aiImporterDesc desc = {
 };
 
 glTFImporter::glTFImporter() :
-        BaseImporter(), meshOffsets(), embeddedTexIdxs(), mScene(nullptr) {
+        mScene(nullptr) {
```

Comment:
base class default c’tors are always called; `std::vector` is always default-initialized